apps/extension/src/components/category_dropdown/CategoryDropDown.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { useState } from 'react';
+// import { Icon } from '@pinback/design-system/icons';
+import { Icon } from '@pinback/design-system/icons';
+import { cva } from 'class-variance-authority';
+
+const categoryDropdownVariants = cva('cursor-pointer shrink-0', {
+  variants: {
+    size: {
+      large: 'w-[32.7rem]',
+      medium: 'w-96',
+    },
+  },
+  defaultVariants: {
+    size: 'large',
+  },
+});
+
+const categoryFontVariants = cva('text-black', {
+  variants: {
+    size: {
+      large: 'body2-m',
+      medium: 'sub7-sb',
+    },
+  },
+  defaultVariants: {
+    size: 'large',
+  },
+});
+interface CategoryDropDownProps {
+  size?: 'large' | 'medium';
+  categories: string[];
+}
+
+const CategoryDropDown = ({ size, categories }: CategoryDropDownProps) => {
+  const getDropdownWidth = ({ size }: { size?: 'large' | 'medium' }) => {
+    return size === 'large' ? { width: '32.7rem' } : { width: '26rem' };
+  };
+  console.log('size', size);
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
드롭다운이 열리고 닫히는 상태에 대한 값이니 `isOpen` 보다 명확한 네이밍으로 수정하는 것이 좋을 것 같아요!

```suggestion
const [isDropdownOpen, setIsDropdownOpen] = useState(false);
```